### ISSUE TITLE
VEVO videos skipped querying a channel

### ISSUE THREAD

**Issue by radiolondra** [Tyrrrz/YoutubeExplode#654]
### Version

ver 6.2.1

### Details

The query of a channel with Channels.GetUploadsAsync() seems to skip all VEVO videos.

### Steps to reproduce

call Channels.GetUploadsAsync() for a channel url (e.g. https://www.youtube.com/channel/UC9Nyyq3qHochdTq1klPyVIQ)

**Comment by radiolondra** [comment]
I checked the Dido channel (the channel I used in my previous post) directly in browser with:
https://www.youtube.com/playlist?list=UU9Nyyq3qHochdTq1klPyVIQ
(note the "UU" at the start of the channel id which should mean "uploaded videos") 
and effectively **Youtube returns the same list without all VEVO videos**. So it's not a bug! Sorry for that.

But now the question is: 
- from where are coming all those VEVO videos in the "Video" tab of the channel? 
- And how can I get the same videos shown in the channel's Video tab?
